### PR TITLE
crow release touchups

### PR DESCRIPTION
**Pull request by dndrks**
general cleanup of the crow docs, ahead of 4.0's release. mostly formatting + page layout, but also changed a few content bits:

- the norns study now has no mention of `ii.pullup`
- the script reference now has default values listed where applicable
- rearranged `update.md` and added 4.0.1 changelog (@trentgill , unsure if what's posted at https://github.com/monome/crow/releases/tag/v4.0.1 is complete or if you wanna massage it, but figured it was fine as a placeholder!)

### CODE REVIEW

File: crow/index.md [hunk 1]
```diff
@@ -38,13 +36,13 @@ If you would like to use the [ii](/docs/modular/ii) functionality, be sure to ob
 
 ### norns
 
-On October 1st 2019, we released a norns [update](../norns/#update) which allows crow to integrate seamlessly as a CV and [ii](/docs/modular/ii) interface. [Many scripts have already been updated to utilize crow](https://llllllll.co/search?expanded=true&q=tags%3Acrow%2Bnorns%20order%3Alatest).
+crow integrates seamlessly with norns as a CV and [ii](/docs/modular/ii) interface. By default, crow can be a norns clocking source or destination. [Many scripts have additional crow functionality](https://llllllll.co/search?expanded=true&q=tags%3Acrow%2Bnorns%20order%3Alatest).
```
Comment (tehn) [inline crow/index.md]: maybe we link to norns.community?

File: crow/index.md [hunk 2]
```diff
@@ -58,37 +56,23 @@ Using the custom `[crow]` object in Max 8, create your own Live-controllable dev
...
-
-Contact *help@monome.org* with further questions.
-
-## Calibration
-
-There are also a subset of commands for managing the state of the device and contents of flash memory. crow ships pre-calibrated, but it is possible to re-run the automatic calibration.
-See the [technical](technical) page for further details.
+If you are writing or modifying norns scripts, standalone druid snippets, or Max patches, you'll want to visit the [scripting reference](reference) to become fluent in the language of the birds.
 
+## Updates + Development
```
Comment (tehn) [inline crow/index.md]: updates and development are pretty disparate topics, so i'd split them up. updates probably doesn't need to mention the github link (it's out of context here).

**Review by tehn — APPROVED** [review]
awesome!! just minor notes

File: crow/index.md (hunk 1 above)
Comment (dndrks) [reply crow/index.md]: ooo great call!

File: crow/index.md (hunk 2 above)
Comment (dndrks) [reply crow/index.md]: super good call, ty!

**Pull request merged**